apps/studio/pages/api/database/[ref]/pg-bouncer.ts

```diff
@@ -0,0 +1,33 @@
+import { NextApiRequest, NextApiResponse } from 'next'
+import apiWrapper from 'lib/api/apiWrapper'
+
+export default (req: NextApiRequest, res: NextApiResponse) => apiWrapper(req, res, handler)
+
+async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { method } = req
+
+  switch (method) {
+    case 'GET':
+      return handleGetAll(req, res)
+    default:
+      res.setHeader('Allow', ['GET'])
+      res.status(405).json({ data: null, error: { message: `Method ${method} Not Allowed` } })
+  }
+}
+
+const handleGetAll = async (req: NextApiRequest, res: NextApiResponse) => {
+  const host = req.query.host
+
+  try {
+    const url = `http://${host}:6543`
```

Comment:
We need to do some validation here to make sure we arent pinging arbitrary hosts and only the pgbouncer hosts belonging to the user/project, else it could lead to SSRF vuln. Given the project ref, we can construct the host directly on the api instead of relying on a separate host query parameter